src/twisted/python/test/test_release.py

```diff
@@ -19,12 +19,14 @@
 import shutil
 
 from io import BytesIO, StringIO
+from unittest import skipIf
 
 from twisted.trial.unittest import TestCase, FailTest, SkipTest
 
 from twisted.python.procutils import which
 from twisted.python import release
 from twisted.python.filepath import FilePath
+from twisted.python.reflect import requireModule
```

Comment:
I think this can be `importlib.import_module` now. But we can save that for another pr

Reply:
I don't see  import_module used anywhere in Twisted.... so yes. maybe update all  requireModule calls in separate PR